cmd/policy-tool/lint/lint.go

```diff
@@ -0,0 +1,33 @@
+package lint
+
+import (
+	"github.com/forseti-security/config-validator/pkg/gcv"
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &cobra.Command{
+	Use:     "lint",
+	Short:   "Lint a directory continaing ConstraintTemplates and/or Constraints.",
+	Example: `policy-tool status --path ./forseti-security/policy-library/policies`,
+	RunE:    lintCmd,
+}
+
+var (
+	flags struct {
+		policies []string
+		libs     string
+	}
+)
+
+func init() {
+	Cmd.Flags().StringSliceVar(&flags.policies, "policies", nil, "Path to the policies directory.")
```

Comment:
this can be > 1 directory right? mention that in the description?

Reply:
done